### ISSUE TITLE
BigInt fails in the constructor

### ISSUE THREAD

**Issue by astojanov** [scala-native/scala-native#327]
Consider the following code:

``` scala
import scalanative.native._
import stdio._

object App {
  def main(args: Array[String]): Unit = {
    val a = BigInt(1)
    fprintf(stdout, c"bigint is: %d\n", (a).toInt)
  }
}
```

it fails as soon as the constructor of `BigInt` is called:

```
> nativeLink
[trace] Stack trace suppressed: run 'last *:nativeLink' for the full output.
[error] (*:nativeLink) scala.scalanative.util.package$UnsupportedException: Is(F(64),Local(Local(src,1),Ptr)) (class scala.scalanative.nir.Op$Is)
```

Looking at the error description, it could be that this issue is related to #326 - and if that is the case, my apologies for the duplicate.

**Comment by densh** [comment]
Thanks!

**Comment by densh** [comment]
Fixed in 6a10bc8
